CONTRIBUTING.md

```diff
@@ -91,16 +91,6 @@ Please refer to the [README](README.md) for source installation and environment
 setup instructions, then please read the [Getting Started
 page](docs/source/get_started/get_started.rst).
 
-### Setting up Git
-
-We use `#` as a special character to denote issue numbers in our commit
-messages. Please change your comment character in your Git to not conflict with
-this:
-
-```
-git config core.commentchar ">"
```

Comment:
❤️

Reply:
I never did this lol ... hope I didn't break anything. It might have been configured globally on my machine.